app/assets/javascripts/Components/Modals/graders_distribution_modal.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Modal from "react-modal";
+import PropTypes from "prop-types";
+
+export class GraderDistributionModal extends React.Component {
+  static defaultProps = {
+    override: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      override: this.props.override,
+    };
+  }
+
+  componentDidMount() {
+    this.props.graders.forEach(grader => {
+      !this.props.weightings[grader._id] ? (this.props.weightings[grader._id] = 1) : null;
+    });
+    Modal.setAppElement("body");
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    this.props.onSubmit();
+  };
+
+  createGraderRow = grader => {
+    return (
+      <div className="flex-row-expand" key={grader.user_name}>
+        <label className="modal-inline-label">{grader.user_name}:</label>
+        <input
+          id={`input-${grader.user_name}`}
+          type="Number"
```

Comment:
I think it's more standard to use lowercase for input types (e.g., `number` instead of `Number`)